app/javascript/components/AdvancedSearch/SetsPicker.jsx

```diff
@@ -1,70 +1,77 @@
-import React from 'react';
+import React from "react";
 
 const SET_DETAILS = {
-  "mkm": {
-    text: "Murders at Karlov Manner",
-    symbol: 'ss ss-mkm'
-  },
-  "plst": {
-    text: "The List",
-    symbol: 'ms ms-planeswalker'
-  },
-  "spg": {
-    text: "Special Guest",
-    symbol: 'ss ss-spg'
-  },
-  "otj": {
-    text: "Outlaws of Thunder Junction",
-    symbol: 'ss ss-otj'
-  },
-  "big": {
-    text: "The Big Score",
-    symbol: 'ss ss-big',
-  },
-  "otp": {
-    text: "Breaking News",
-    symbol: 'ss ss-otp',
-  },
-  "mh3": {
-    text: "Modern Horizons 3",
-    symbol: 'ss ss-mh3',
-  },
-  "m3c": {
-    text: "Modern Horizons 3 Commander",
-    symbol: 'ss ss-m3c',
-  }
-}
+    mkm: {
+        text: "Murders at Karlov Manner",
+        symbol: "ss ss-mkm",
+    },
+    plst: {
+        text: "The List",
+        symbol: "ms ms-planeswalker",
+    },
+    spg: {
+        text: "Special Guest",
+        symbol: "ss ss-spg",
+    },
+    otj: {
+        text: "Outlaws of Thunder Junction",
+        symbol: "ss ss-otj",
+    },
+    big: {
+        text: "The Big Score",
+        symbol: "ss ss-big",
+    },
+    otp: {
+        text: "Breaking News",
+        symbol: "ss ss-otp",
+    },
+    mh3: {
+        text: "Modern Horizons 3",
```

Comment:
@blseich should we clean these old ones out

Reply:
We can keep them for now and clean them up later... either way works. It will only pull in the set codes that are actually being used anyway so it isn't dire. Once those cards are out of the DB those won't appear on the page anymore.